src/libcore/num/f32.rs

```diff
@@ -474,4 +474,65 @@ impl f32 {
         // It turns out the safety issues with sNaN were overblown! Hooray!
         unsafe { mem::transmute(v) }
     }
+
+    #[unstable(feature = "float_to_from_bytes", issue = "0")]
+    #[rustc_const_unstable(feature = "float_to_from_bytes_const")]
+    #[inline]
+    pub const fn to_be_bytes(self) -> [u8; 4] {
+        swap_if_le(self.to_ne_bytes())
```

Comment:
We already have `f32::to_bits()` that gives `u32`; I'd expect this to just be `self.to_bits().to_ne_bytes()` (and analogously elsewhere).